### ISSUE TITLE
Daily Chart does not grab entered data.

### ISSUE THREAD

**Issue by Show-M** [weaveio/forum#131]
<img width="1024" alt="スクリーンショット 2021-05-30 11 28 06" src="https://user-images.githubusercontent.com/61481039/120090039-6fa0b880-c13a-11eb-8c89-71ef46341e62.png">

<br>
<br>
DataSource:
<img width="1288" alt="スクリーンショット 2021-05-30 11 29 33" src="https://user-images.githubusercontent.com/61481039/120090030-66175080-c13a-11eb-9ce3-93375b55c026.png">

```
    {
      "cols": [
        {
          "tile": "chartSparse",
          "title": "日平均",
          "display": {
            "span": "12"
          },
          "chart": {
            "timeSeries": "YYYY-MM-DD; YYYY-MM-DD; P1D",
            "title": "トップフロー値",
            "c3": {
              "data.type": "line",
              "size.height": 200,
              "axis.y.show": true,
              "xxxdata.labels": true,
              "axis.x.tick.format": "%b %d",
              "legend.show": true,
              "color.pattern": [
                "#0095FF",
                "#FF3DF5"
              ]
            }
          },
          "connect": [
            {
              "title": "朝",
              "source": "top",
              "pick": "values.topPeakflowMorning",
              "apply": "mean(ls)",
              "format": "0,0",
              "tableData": {
                "id": 1
              }
            },
            {
              "title": "夜",
              "source": "top",
              "pick": "values.topPeakflowNight",
              "apply": "mean(ls)",
              "format": "0,0",
              "tableData": {
                "id": 2
              }
            }
          ]
        }
      ]
    },
```

**Comment by rgagne** [comment]
Problem with picker evaluation; fixed and deploying (will take 20-30 min before live)

**Comment by Show-M** [comment]
<img width="935" alt="スクリーンショット 2021-05-30 18 27 45" src="https://user-images.githubusercontent.com/61481039/120099123-c32ff800-c174-11eb-8a72-21e94dee98a9.png">

**Comment by Show-M** [comment]
@rgagne 
Not sure what happened...but it is not working again. Can you kindly have a look?

**Comment by rgagne** [comment]
it is fixed, make sure you're on the right version

**Comment by Show-M** [comment]
It was fixed till when i QAed last night. 
Now it is fixed again on 3.0.2643
